modular_skyrat/modules/clock_cult/code/structures/traps/trap.dm

```diff
@@ -0,0 +1,114 @@
+//Thing that you stick on the floor
+/obj/item/clockwork/trap_placer
+	name = "trap"
+	desc = "don't trust it"
+	icon = 'modular_skyrat/modules/clock_cult/icons/clockwork_objects.dmi'
+	w_class = WEIGHT_CLASS_HUGE
+	var/result_path = /obj/structure/destructible/clockwork/trap
+
+/obj/item/clockwork/trap_placer/attack_self(mob/user)
+	. = ..()
+	if(!(FACTION_CLOCK in user.faction))
+		return
+	for(var/obj/structure/destructible/clockwork/trap/trap in get_turf(src)) // No 50-spear instakills please
+		if(!istype(trap, type))
```

Comment:
Is this meant to be `result_path` rather than `type`? otherwise this istype() will always be false. For that matter, I'm not fully sure if it's good to allow multiple traps, each a different type, on the same tile, unless that's an intentional feature.

Reply:
It'll nerf traps pretty hard if you can't put a pressure plate on the same tile as a trap; might redo it later to only allow for one *damaging* trap, but it can still do the regular amount of triggers